src/python/WMCore/WMBS/Oracle/WorkUnit/Add.py

```diff
@@ -0,0 +1,34 @@
+"""
+Oracle implementation of Add WorkUnit
+"""
+from WMCore.WMBS.MySQL.WorkUnit.Add import Add as AddWorkUnitMySQL
+
+
+class Add(AddWorkUnitMySQL):
+    """
+    _Add_
+
+    Use Oracle sequence instead of auto_increment and do it in a combined transaction
+    """
+
+    sql = ('INSERT ALL'
```

Comment:
@hufnagel I think this is what you were talking about? I looked at your example and you were using SQL to check for existence and return the inserted values as well, but I think this is the heart of "InsertStreamer"